### ISSUE TITLE
Enter serial number manually button should be available all the time during camera scan of BCSC 

### ISSUE THREAD

**Issue by knguyenBC** [bcgov/bc-wallet-mobile#3969]
### Description of feature / user story
As a user who is unable to scan my BCSC (accessibility)
I want to be able to enter my serial number manually
So that I can continue the verification process

### Acceptance Criteria
Given the user is prompted to scan a BCSC
When the camera is prompted
Then the "Enter Serial Number Manually" button should immediately be present
And the button design should be the secondary button design on dark background

### Wireframes or relevant image assets / links

TBD
